Makefile

```diff
@@ -152,8 +152,8 @@ install.completions:
 
 install.systemd:
 	install -D -m 644 contrib/systemd/crio.service $(PREFIX)/lib/systemd/system/crio.service
+	install -D -m 644 contrib/systemd/crio.service $(PREFIX)/lib/systemd/system/cri-o.service
```

Comment:
Actually, isn't this the same as the line above it?

Reply:
Nope it has a dash after cri in cri-o

Reply:
We end up with two services this way where the original was just one, why not just fix to:
```
 -	ln -s crio.service $(PREFIX)/lib/systemd/system/cri-o.service 
 +	ln -sf crio.service $(PREFIX)/lib/systemd/system/cri-o.service 
```

Would that solve your issue?

Reply:
@runcom ????

Reply:
I'm fine either way actually